tools/wptrunner/wptrunner/executors/executoratspi.py

```diff
@@ -4,13 +4,30 @@
 from gi.repository import Atspi
 import json
 import threading
+import time
+
+import sys
+
+def poll_for_active_tab(root):
+    active_tab = find_active_tab(root)
+    iterations = 0
+    while not active_tab:
+        time.sleep(0.01)
+        active_tab = find_active_tab(root)
+        iterations += 1
```

Comment:
Ah! I didn't quite understand when you explained this morning -- you literally are polling to see if the active tab is there, not polling the events. So we no longer need to wait for events at all, we could do this on the other platforms as well.